### PR TITLE
Documentación asistente para listar pólizas sin contrataciones

### PR DESCRIPTION
**Pull request by polsala**
### Description:

Added documentation in distri/contrataciones for listing contracts without engagement.

### Tasks:

- [x] Documentation
- [x] Translation
- [x] Images

### Links:

* [facturacion_contratacion.md #Assistent per llistar pòlisses sense contractacións (sense facturació de drets)](http://builds.gisce.net/powerp-docs/powerp_docu_for_listing_contracts_with_no_engagement/ca/distri/facturacion_contratacion/#assistent-per-llistar-polisses-sense-contractacions-sense-facturacio-de-drets)
* [facturacion_contratacion.md #Asistente para listar pólizas sin contrataciones (sin facturación de derechos)](http://builds.gisce.net/powerp-docs/powerp_docu_for_listing_contracts_with_no_engagement/es/distri/facturacion_contratacion/#asistente-para-listar-polizas-sin-contrataciones-sin-facturacion-de-derechos)

Documents gisce/erp#5592

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_listing_contracts_with_no_engagement/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/facturacion_contratacion.md [hunk 1]
```diff
@@ -174,3 +174,19 @@ factura generada altrament es tancarà l'assistent i la pòlissa ja estarà reco
 ![](_static/facturacion_contratacion/w_4.png)
 
 ![](_static/facturacion_contratacion/w_5.png)
+
+### Assistent per llistar pòlisses sense contractacións (sense facturació de drets)
```
Comment (jaumef) [inline docs/distri/facturacion_contratacion.md]: `contractacións` → `contractacions`

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_listing_contracts_with_no_engagement/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/facturacion_contratacion.md [hunk 2]
```diff
@@ -174,3 +174,19 @@ factura generada altrament es tancarà l'assistent i la pòlissa ja estarà reco
...
+Aquest assistent permet llistar totes les pòlisses que no tenen cap contractació.
+
+#### Llistar pólisses sense contractacions
+
+L'assistent el podem localitzar en el menú genèric de pòlisses, en el botó
+**Contractes sense facturació de drets**.
+
+![](_static/facturacion_contratacion/wiz_contracts_no_engangement_menu.png)
+
+Al prémer el botó, automaticament se'ns llistaran totes les pólisses que no
```
Comment (jaumef) [inline docs/distri/facturacion_contratacion.md]: `Al prémer el botó, automaticament se'ns llistaran` → `Ens llistarà`

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_listing_contracts_with_no_engagement/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/facturacion_contratacion.md [hunk 3]
```diff
@@ -174,3 +174,18 @@ factura generada altrament es tancarà l'assistent i la pòlissa ja estarà reco
...
+Aquest assistent permet llistar totes les pòlisses que no tenen cap contractació.
+
+#### Llistar pólisses sense contractacions
+
+L'assistent el podem localitzar en el menú genèric de pòlisses, en el botó
+**Contractes sense facturació de drets**.
+
+![](_static/facturacion_contratacion/wiz_contracts_no_engangement_menu.png)
+
+Ens llistara totes les pólisses que no tinguin contractacions.
```
Comment (gdalmau) [inline docs/distri/facturacion_contratacion.md]: `llistara` → `llistarà`
`pòlisses`→ `pòlisses`

File: docs/distri/facturacion_contratacion.md [hunk 4]
```diff
@@ -174,3 +174,18 @@ factura generada altrament es tancarà l'assistent i la pòlissa ja estarà reco
 ![](_static/facturacion_contratacion/w_4.png)
 
 ![](_static/facturacion_contratacion/w_5.png)
+
+### Assistent per llistar pòlisses sense contractacions (sense facturació de drets)
+
+Aquest assistent permet llistar totes les pòlisses que no tenen cap contractació.
+
+#### Llistar pólisses sense contractacions
```
Comment (gdalmau) [inline docs/distri/facturacion_contratacion.md]: `pòlisses`→ `pòlisses`

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_listing_contracts_with_no_engagement/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_listing_contracts_with_no_engagement/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by jaumef** [comment]
@polsala solve conflicts and we can merge

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_listing_contracts_with_no_engagement/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_listing_contracts_with_no_engagement/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_listing_contracts_with_no_engagement/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**